client-httpurlconnection/build.gradle

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'jacoco'
+apply plugin: 'jacoco-android'
+
```

Comment:
Let's use a consistent prefix for our modules (This will make it easier to publish them later):

* fretboard
* fretboard-storage-flatfile (AtomicFile is an implementation detail and _could_ change).
* fretboard-client-kinto

Reply:
Done